python/tvm/relay/op/nn/_nn.py

```diff
@@ -548,3 +548,13 @@ def schedule_deformable_conv2d(attrs, outs, target):
 
 
 reg.register_pattern("nn.deformable_conv2d", OpPattern.OUT_ELEMWISE_FUSABLE)
+
+reg.register_schedule("nn.cross_entropy", schedule_injective)
+
+reg.register_pattern("nn.cross_entropy",
+                     OpPattern.OPAQUE)
+
+@reg.register_compute("nn.cross_entropy")
+def compute_cross_entropy(attrs, inputs, out_dtype, target):
+    x, y = inputs
+    return [-topi.sum(topi.nn.log_softmax(x) * y / x.shape[0])]
```

Comment:
This looks like it's doing fused softmax+cross-entropy from logits. For a normal feed-forward model that's converted into relay, the outputs will already be softmax; do you have an example of how you'd remove that softmax layer and add this cross-entropy from logits layer instead?

Reply:
I havent thought that much when I was doing this - I am only trying to get an end to end training example.